metaflow/__init__.py

```diff
@@ -164,8 +161,13 @@ class and related decorators.
             alias_submodules(extension_module, tl_package, None, extra_indent=True)
         )
         version_info = getattr(extension_module, "__mf_extensions__", "<unk>")
-        if extension_module.__version__:
-            version_info = "%s(%s)" % (version_info, extension_module.__version__)
+        ext_version = _format_git_describe(
+            _call_git_describe(cwd=os.path.dirname(extension_module.__file__))
+        )
+        if ext_version is None:
```

Comment:
shouldn't this be the other way around?

also, i am getting `Metaflow 2.12.18.post5-git2201f4e-dirty+ob(4.3.19) executing...` - the interesting thing is that we don't have version 4.3.19 for any extensions internally. we pin the extension version explicitly to v1. it's likely that `git describe --tags` is the offender here.

Reply:
@madhur-ob do you happen to know how ob releases are git tagged?